### ISSUE TITLE
converting saved model to TFlite format 

### ISSUE THREAD

**Issue by enkaranfiles** [AlexeyAB/darknet#5262]
I would like to convert custom yolo model to TFlite format to purpose of detecting potholes on the roads with Flutter application. When I tried darkflow to converting model into pb graphdef file after converting tflite format, model has spoiled. any ideas? any way to convert tflite format?

Thank you for your time!!

**Comment by AlexeyAB** [comment]
Did you try to convert Darknet model to TensorFlow pb-model? https://github.com/AlexeyAB/darknet/wiki/Converting-Yolo-v3-models-to-TensorFlow-and-OpenVINO(IR)-models

Did you try to convert pb-model to Tflite model? https://www.tensorflow.org/lite/convert/cmdline_examples

**Comment by enkaranfiles** [comment]
yes, I have tried. But when I convert into pb file I have used another framework darkflow.
is that matter?

**Comment by enkaranfiles** [comment]
Also the link that you have shared can also be applicable on tiny-yolov2?
